extensions/amp-access/0.1/access-expr-impl.md

```diff
@@ -0,0 +1,39 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## Overview
+
+This documentation describes how to build `access-expr-impl.js` based on the
+`access-expr-impl.jison`.
+
+We use [Jison](https://zaach.github.io/jison/) grammar compiler.
+
+## Prerequisities
+
+Install Jison:
+
+```
+npm install jison -g
```

Comment:
I don't expect this to change much, so maybe it doesn't matter, but have you thought about using https://www.npmjs.com/package/gulp-jison and not check in the JS file?